### ISSUE TITLE
Activation missing after: h = keras.layers.Conv2D(1, 4, strides=1, padding='same', ...)(h)

### ISSUE THREAD

**Issue by TechNilabh** [ComputerScienceSoceityNITS/Open-Source-Marathon-2025#9]
Final activation missing after: h = keras.layers.Conv2D(1, 4, strides=1, padding='same', ...)(h). Choose a proper activation and address this issue. Justify youer approach with output.
